resources/views/partials/navbar/menu-item-search-form.blade.php

```diff
@@ -1,6 +1,6 @@
-<form action="{{ $item['href'] }}" method="{{ $item['method'] }}" class="form-inline mx-2">
+<form action="{{ $item['href'] }}" method="{{ $item['method'] }}" class="form-inline ml-3">
```

Comment:
`ml-3` will be not adequate for the search bar if you configure it for the right section (using `topnav_right=true`). AdminLTE site is just a reference, however we can try reducing to `mx-1`.